### ISSUE TITLE
Adapt the LICENSE renaming in order to account for the file extension

### ISSUE THREAD

**Issue by EnnuiL** [QuiltMC/quilt-example-mod#19]
On QTM, `LICENSE.md` is to be considered a valid alternative to the good ol' `LICENSE`, but if you do try it, the renamed LICENSE will become something like `LICENSE.md_example_mod`, which is bad

**Comment by triphora** [comment]
README doesn't get included in the jar by default...
Besides, you don't have to rename it, you just do `from 'README.md'` and you're done

**Comment by EnnuiL** [comment]
> README doesn't get included in the jar by default... Besides, you don't have to rename it, you just do `from 'README.md'` and you're done

The buildscript does include it, although the README rename is something inherited from Fabric Example Mod, which I assume it's for clarifying what the LICENSE file belongs to; Although thinking about it? It's only really relevant if you actually include more than one README file which in that case, yeah, it should be added manually

**Comment by triphora** [comment]
> The buildscript does include it,

Show me where it does that. [It doesn't.](https://github.com/QuiltMC/quilt-template-mod/blob/1.19/build.gradle#L66-L71)

**Comment by EnnuiL** [comment]
> > The buildscript does include it,
> 
> Show me where it does that. [It doesn't.](https://github.com/QuiltMC/quilt-template-mod/blob/1.19/build.gradle#L66-L71)

Yes, like I said, the buildscript does include the LICENSE file, and I don't see why we wouldn't do that

**Comment by EnnuiL** [comment]
...🤦‍♀️ I failed at communication; When I said `README`, I meant `LICENSE`; i sure love my brain

**Comment by EnnuiL** [comment]
Unfortunately, the attempted solution failed... It has been reverted for now, but yeah, an equivalent one that actually works should be investigated
